core/process/aru.php

```diff
@@ -259,7 +259,7 @@
 	
 	
 	case 'gym_map':
-		$req 		= "SELECT gym_id, team_id, guard_pokemon_id, gym_points, latitude, longitude, (last_modified ".$time->symbol." INTERVAL ".$time->delay." HOUR) as last_modified FROM gym";
+		$req 		= "SELECT gym_id, team_id, guard_pokemon_id, gym_points, latitude, longitude, (CONVERT_TZ(last_modified, '+00:00', '".$time_offset."')) as last_modified FROM gym";
```

Comment:
Why do we have to ``CONVERT_TZ(...`` here? I have my worries that this might also have some issues like the ``DATE(...`` we used before.